### ISSUE TITLE
Windows build is broken

### ISSUE THREAD

**Issue by xenu** [houseabsolute/ci-perl-helpers#8]
It fails in "run tests" section:
```
##[section]Starting: Run tests
==============================================================================
Task         : Bash
Description  : Run a Bash script on macOS, Linux, or Windows
Version      : 3.159.3
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/bash
==============================================================================
Generating script.
[command]"C:\Program Files\Git\bin\bash.exe" --noprofile --norc -c pwd
/d/a/_temp

========================== Starting Command Output ===========================
[command]"C:\Program Files\Git\bin\bash.exe" --noprofile --norc /d/a/_temp/3bde01f0-2b4e-4aeb-b028-75b711860689.sh
+ 'd:\a\1\ci-perl-helpers-tools\bin\with-perl' 5.30.0_64 run-tests.pl
+ berrybrew exec --with 5.30.0_64 perl 'd:\a\1/ci-perl-helpers-tools/bin/run-tests.pl'
mkdir C:\Users\VSSADM~1\AppData\Local\Temp\MRL78LqSbf\D:\: Invalid argument; The filename, directory name, or volume label syntax is incorrect at C:/berrybrew/runtime-perl/perl/lib/TAP/Harness.pm line 887.
 at C:/berrybrew/runtime-perl/perl/lib/App/Prove.pm line 548.
Running tests with prove failed (exit code = 1). at d:/a/1/ci-perl-helpers-tools/bin/../lib/T/RunTests.pm line 75.

##[error]Bash exited with code '1'.
##[section]Finishing: Run tests
```

Note that reverting to v0.0.3 makes the error go away, so this is a new issue.

**Comment by autarch** [comment]
Yep, I'm working on a fix for this in the `better-tools-handling` branch.

**Comment by autarch** [comment]
This should be fixed in the v0.0.5 tag.
